source/api-docs.txt

```diff
@@ -0,0 +1,72 @@
+.. _rm-rest-api:
+
+============================
+Relational Migrator REST API
+============================
+
+.. facet::
+   :name: genre 
+   :values: tutorial
+
+.. meta:: 
+   :description: Learn about the MongoDB Relational Migrator REST API and how to use it.
+   :keywords: data migration, rest api, open api
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The Relational Migrator REST API provides you with programmatic access 
+to Relation Migrator with HTTPS endpoints. The REST API is a middleware 
+service that sits between your application and your source and destination 
+databases that you can send requests to facilitate and manage your sync 
+jobs.
+
+You don't need to install any additional extensions or packages to use 
+the Relation Migrator REST API. The functionality is built into the core 
+Relational Migrator product and runs on the same port that you access 
+the application on (either ``8278``, ``8080``, or ``443``).
+
+How the REST API Works
+----------------------
+
+The Relational Migrator REST API:
+
+- Runs on the machine or server that the Relation Migrator application
+  is installed on.
+- The Rest API is governed by the host adapter binding.
+- The Relational Migrator REST API **does not** currently support 
+  authentication.
+
+When to Use REST API
+--------------------
+
+We recommend using the Relational Migrator REST API when:
+
+- You want to use a task scheduler such as 
+  `CRON <https://en.wikipedia.org/wiki/Cron>`__ and define dynamic 
+  :ref:`table filters <rm-table-filters>` within a project such as 
+  ``WHERE DATEDIFF(CURDATE(), last_modified) <= 1`` to batch migrate 
+  data on a cadence from your relational database to MongoDB. 
+
+- You want to programmatically create Relational Migrator sync jobs 
+  quickly and dynamically. For example, using connection strings from 
+  multiple databases without manually configuring them in the GUI.
+
+- You generally prefer scripting your workflows instead of using a GUI.
+
+Constraints
+-----------
+
+The following constraints apply to Data API requests and responses: 
+
+- Requests will time out after 300 seconds of runtime.
+
+- API response payloads are limited to 350 MB in size.
```

Comment:
Borrowed these from the atlas docs, please LMK if there are additional constraints.

Reply:
I would skip this the mention of payload size; while I'm sure it's technically true there is no reason the payloads would be anywhere near this size for this API.